test/avocado/selenium-machines-basic.py

```diff
@@ -107,3 +108,43 @@ def testDelete(self):
         self.assertNotIn(name, self.machine.execute("sudo virsh list --all"))
         self.assertNotIn(imgdel, self.machine.execute("sudo virsh vol-list {}".format(args.get('poolName'))))
         self.assertIn(args.get('image'), self.machine.execute("sudo virsh vol-list {}".format(args.get('poolName'))))
+
+    def testVmStatus(self):
+        name = 'staticvm'
+        self.create_vm(name)
+
+        self.assertEqual(
+            self.machine.execute('sudo virsh domstate {}'.format(name)).replace("\n",""),
```

Comment:
You can use rstrip to remove trailing newlines https://docs.python.org/3/library/stdtypes.html#str.rstrip

Reply:
Fixed in commit